CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
+* Mark social sharing images as aria-hidden ([PR #1633](https://github.com/alphagov/govuk_publishing_components/pull/1633))
```

Comment:
I don't think we need to mention this in the changeling since it's an internal change and it's documented in the commit log.

Reply:
👍